cop-import-export.php

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Not semantically, i did that for align the buttons of import and export, the right should be <form><p>...</p></form>, my bad